frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -342,6 +343,16 @@ function ReaderHighlight:onHoldPan(_, ges)
     UIManager:setDirty(self.dialog, "ui")
 end
 
+local info_message_ocr_text = _([[
+No OCR results or no language data.
+
+KOReader has a build-in OCR engine for recognizing words in scanned PDF and DjVu documents. In order to use OCR in scanned pages, you need to install tesseract trained data for your document language.
+
+You can download language data files from https://github.com/tesseract-ocr/tesseract/wiki/Data-Files#data-files-for-version-304305
```

Comment:
@poire-z I don't really like the anchor (makes it seem like you might have to type in the whole thing) but at the same time it might be a bit unclear without. :-/

Reply:
You could just say:
`You can download language data files for version 3.04 from https://github.com/tesseract-ocr/tesseract/wiki/Data-Files`

(We'll have to think about updating this text on next tesseract updates)